benefits/core/session.py

```diff
@@ -53,7 +53,7 @@ def context_dict(request):
         _DEBUG: debug(request),
         _DID: did(request),
         _FLOW: flow(request),
-        _ELIGIBILITY: eligibility(request),
+        _ELIGIBILITY: eligible(request),
```

Comment:
I think we should rename this constant to `_ELIGIBLE` for consistency.

Reply:
Yep, renamed it [here](https://github.com/cal-itp/benefits/blob/903f76f7bf932cf401dba9e78887ee6b2da5c1b6/benefits/core/session.py#L22) and everywhere else it is used